test/n2z/N2Z_Regression.ipynb

```diff
@@ -0,0 +1,230 @@
+    "* Use the `run_notebooks.sh` script, which requires you to declare an environment variable `HARMONY_HOST_URL`. Set that environment variable to the value above that corresponds to the environment you want to test. That environment variable will take precedence over the default value in the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7b732c0",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "harmony_host_url = 'https://harmony.uat.earthdata.nasa.gov'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4979305a",
+   "metadata": {},
+   "source": [
+    "### Import required packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93128635",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from harmony import Collection, Environment, Client, Request\n",
+    "from os import makedirs\n",
+    "\n",
+    "from utility import assert_result_has_correct_number_of_stores, print_success"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e29b32f",
+   "metadata": {},
+   "source": [
+    "### Set up environment dependent variables:\n",
+    "\n",
+    "This includes the Harmony `Client` object and `Collection` objects for each of the collections for which there are regression tests. The local, SIT and UAT Harmony instances all utilise resources from CMR UAT, meaning any non-production environment will use the same resources.\n",
+    "\n",
+    "When adding a production entry to the dictionary below, the collection instances can be included directly in the production dictionary entry, as they do not need to be shared.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5c95750",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "non_production_collection = {\n",
+    "    'imerg_collection': Collection(id='C1245618475-EEDTEST'),\n",
+    "}\n",
+    "\n",
+    "collection_data = {\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': {\n",
+    "        **non_production_collection,\n",
+    "        'env': Environment.UAT\n",
+    "    },\n",
+    "    'https://harmony.sit.earthdata.nasa.gov': {\n",
+    "        **non_production_collection,\n",
+    "        'env': Environment.SIT\n",
+    "    },\n",
+    "    'http://localhost:3000': {\n",
+    "        **non_production_collection,\n",
+    "        'env': Environment.LOCAL\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "environment_information = collection_data.get(harmony_host_url)\n",
+    "\n",
+    "if environment_information is not None:\n",
+    "    harmony_client = Client(env=environment_information['env'])\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae80e3cf",
+   "metadata": {},
+   "source": [
+    "## Test for a single, non-aggregated granule input"
```

Comment:
This might be redundant (feel free to push back on this), but it might be nice to have a description of the expected test outputs in each markdown block here.

That said, the `print_success` statements in the tests give a fairly accurate summary of what was expected.